lib/rules/function-url-quotes/index.js

```diff
@@ -74,7 +73,7 @@ const rule = function(expectation, options) {
 			const hasQuotes = leftTrimmedArgs[0] === "'" || leftTrimmedArgs[0] === '"';
 
 			const trimmedArg = args.trim();
-			const isEmptyArgument = _.includes(['', "''", '""'], trimmedArg);
+			const isEmptyArgument = ['', "''", '""'].includes(trimmedArg);
```

Comment:
It's not relevant to this PR, but I think it's worth changing for readbility:

```suggestion
			const isEmptyArgument = [``, `''`, `""`].includes(trimmedArg);
```

Reply:
I'd feel better if this was made separately. BTW let me know when you are done so that I clean up the branch (I was rebasing because I wasn't completely done and forgot to mark the PR as draft :P)

Reply:
Don't rebase anymore, please. Because I reviewed already, and will have to review again if you rebase.

Reply:
I'm done as a first pass, all good. I updated #4412 with a list of functions for later